python/src/StandardSpaceCrossEntropyImportanceSampling_doc.i.in

```diff
@@ -0,0 +1,99 @@
+%feature("docstring") OT::StandardSpaceCrossEntropyImportanceSampling
+"Standard Space Cross-Entropy Importance Sampling.
+
```

Comment:
I would suggest to consider the notations introduced in the theory page for [importance simulation](https://openturns.github.io/openturns/latest/theory/reliability_sensitivity/importance_simulation.html). Using the same notations will allow to point to this page while keeping the content as consistent as possible.

Reply:
Thank you for this pointer. This is very challenging because it also impacts all the doc of [NAIS](https://openturns.github.io/openturns/latest/user_manual/_generated/openturns.NAIS.html#openturns.NAIS)